src/mono/wasm/templates/templates/console/main.mjs

```diff
@@ -23,4 +23,4 @@ const exports = await getAssemblyExports(config.mainAssemblyName);
 const text = exports.MyClass.Greeting();
 console.log(text);
 
-await runMainAndExit(config.mainAssemblyName, ["dotnet", "is", "great!"]);
+await runMainAndExit(config.mainAssemblyName, ['dotnet', 'is', 'great!']);
```

Comment:
Shouldn't this be passing the command line arguments from runArgs?

Reply:
Opened https://github.com/dotnet/runtime/issues/76201 for this. The tests are failing because they try to patch this file, and have a hardcoded string:
`.Replace("[\"dotnet\", \"is\", \"great!\"]", "(await `

Reply:
I have updated the test.

Reply:
We can do `process.argv.slice(2)` instead